src/elements/line.ts

```diff
@@ -29,15 +33,20 @@ export default class LineElement {
         this.dash = dash
         this.head = head
         this.tail = tail
+
+        // omitting cap if uknown value
+        // assumed 'sq' when empty
+        // https://c-rex.net/projects/samples/ooxml/e1/Part4/OOXML_P4_DOCX_ln_topic_ID0EJR3IB.html#topic_ID0EJR3IB
+        if (CAP_VALUES.includes(cap)) this.cap = cap
```

Comment:
could you keep it empty if it is? (other options tend to follow this - unless our defaut is different than the spec). Here especially as it will be an issue if you do not specify it (i.e. you will have `cap=""`)

Reply:
PowerPoint adds the value regardless, hence I followed the same mindset.

However omitting the value when not specified seems to be working fine too, according to my tested. 

Another commit with the change is pushed.